src/core/importType.js

```diff
@@ -49,6 +49,10 @@ function isRelativeToSibling(name) {
   return name.indexOf('./') === 0
 }
 
+export function isPrivateModule(name) {
+  return name.indexOf('@') === 0
```

Comment:
not if it's inside `node_modules` - anyone can `mkdir` on disk inside `node_modules` and `require` will respect that.